uint16 rightIR
uint16 midIR
uint16 leftIR
uint8 FSR
